packages/p/plasma-workspace-wallpapers/package.yml

```diff
@@ -0,0 +1,22 @@
+name       : plasma-workspace-wallpapers
+version    : 5.27.10
+release    : 1
+source     :
+    - https://download.kde.org/stable/plasma/5.27.10/plasma-workspace-wallpapers-5.27.10.tar.xz : 82862d512993b958f7f96e191ce951ff2748012292941f73cdd308f37e2e64d0
+homepage   : https://kde.org/plasma-desktop
+license    :
+    - GPL-2.0
+    - LGPL-3.0
+component  : desktop.kde
+summary    : Wallpapers for Plasma Workspaces
+description: |
+    Wallpapers for Plasma Workspaces
+builddeps  :
+    - extra-cmake-modules
+    - qt5-base-devel
```

Comment:
We use a pkgconfig when available `pkgconfig(Qt5Core)` and deps are arranged alphabetically with pkgconfig's going first.